recipes/root5/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "root5" %}
+{% set version = "5.34.36" %}
+{% set sha256 = "fc868e5f4905544c3f392cc9e895ef5571a08e48682e7fe173bd44c0ba0c7dcd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: root_v{{version}}.source.tar.gz
+  url: https://root.cern.ch/download/root_v{{version}}.source.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+   number: 0
+   skip: true  # [win]
+   skip: true  # [py3k]
+
+requirements:
+ 
+ build:
+     - toolchain
+     - gcc  # [linux]
+     - python x.x
+     - gsl
+     - fftw
+     - cmake
+     - cfitsio
+     - xorg-libxft  # [linux]
+     - xorg-libxpm  # [linux]
+     - xorg-libsm  # [linux]
+     - jpeg
+     - libpng
+     - zlib
+     - libxml2
+     - mesalib
```

Comment:
Please update these according to the [global pinnings]( https://github.com/conda-forge/conda-forge.github.io/blob/6bc413f9de7957194afdc239e1f89ba32b549da8/scripts/pin_the_slow_way.py#L39 ).

Reply:
I'm not sure what you mean by that. 

Should I change for example 'libpng' -> 'libpng >=1.6.32,<1.6.35'?

Reply:
As an example, yes. There are several things that need to be pinned though. Would take a look at the list and compare.

Once we switch to `conda-build` 3, the pinning process will be automated. For now we are stuck manually doing these pinnings a little longer.